core/example/tutorial/01_hello_world/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
-  add_executable(HelloWorld hello_world.cpp)
-  target_link_libraries(HelloWorld cabanacore)
-  add_test(NAME Core_tutorial_01 COMMAND ${NONMPI_PRECOMMAND} HelloWorld)
+# the next for lines are just, so that we can use this as
+# a standalone CMakeLists.txt for testing of the Target export
+cmake_minimum_required(VERSION 3.9)
```

Comment:
Should we be setting the project name as well?

Reply:
I was surprised we didn't get a warning from CMake when I looked at the logs on TravisCI.  I guess you can ignore then.

Reply:
Do we need to up the minimum required?

Reply:
We could set the project name, but it is not necessary, `cmake_minimum_required()` is only thing needed.

Reply:
@sslattery this is the minimum required version for the dependent package.